### PR TITLE
Create c increment and decrement operaters

### PR DESCRIPTION
**Pull request by anchal071**
coding mainly using increment and decremant  operaters...

### CODE REVIEW

File: c increment and decrement operaters [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include<stdio.h>
+int main ()
+{
+    int a=10,b =100
```
Comment (Anjali-1801) [inline c increment and decrement operaters:4]: Use Semicolon

File: c increment and decrement operaters [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    float c=10.5, d=100.5
```
Comment (Anjali-1801) [inline c increment and decrement operaters:5]: Use Semicolon

**Review by Anjali-1801 — COMMENTED** [review]
Semicolons are missing at a few places.

**Comment by anchal071** [comment]
Thanks for explaining my mistakes. But I have no idea,  that how to right
this...

File: c increment and decrement operaters (hunk 1 above)
Comment (Vaishnavi-iit) [reply c increment and decrement operaters:4]: Anchal,just go through the changes.You have just left semicolon at 2 places.

**Pull request merged**